include/exadg/operators/operator_base.cpp

```diff
@@ -1031,7 +1032,7 @@ OperatorBase<dim, Number, n_components>::cell_loop_dbc(
   for(auto cell = range.first; cell < range.second; ++cell)
   {
     this->reinit_cell(cell);
-
+    // TODO
     integrator->read_dof_values_plain(src);
```

Comment:
I think we need a separate `IntegratorCell` here, initialized with `dof_index_inhomogeneous`. The problem is that `reinit_cell()` in the line above is hard-coded for the member variable `integrator`, which is initialized with the standard `dof_index`. Hence, issue #507 needs to be addressed.